nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/WEB-INF/partials/processor-details.jsp

```diff
@@ -103,6 +103,18 @@
                         <div class="clear"></div>
                     </div>
                     <div class="setting">
+                        <div id="details-execution-node-container" class="execution-node-setting">
```

Comment:
No worries. I don't think there are a lot of problems with the code. I think what was provided is quite good and really close. Most comments are mostly based on the same premise of retaining the existing configuration until a user has explicitly configured a Processor using the new options. I just tried to mark each spot where we should consider that.

Either way, I am happy to continue working with you on this PR or I would be happy to pick up at this point if you'd prefer.

Thanks again!